src/include/processor/operator/index_lookup.h

```diff
@@ -16,14 +17,13 @@ struct IndexLookupInfo {
     std::shared_ptr<BatchInsertSharedState> batchInsertSharedState;
     DataPos keyVectorPos;
     DataPos resultVectorPos;
+    std::vector<DataPos> warningDataPos;
```

Comment:
if we are calling it as `xxVectorPos`, it's better to follow the naming rather than calling it `xxxDataPos`

Reply:
updated to `warningDataVectorPos`